### PR TITLE
Footer redesign

### PR DESCRIPTION
**Pull request by estacioneto**
#### What is the purpose of this pull request?
The footer was previously designed, but not fully studied. Now, our designer has a solution.

#### What problem is this solving?
Footer not designed based in stores.

#### How should this be manually tested?

[Access the workspace](https://estacio--storecomponents.myvtex.com/admin/cms/storefront)

#### Desktop
image

#### Mobile
image

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Footer/global.css [hunk 1]
```diff
@@ -24,6 +29,13 @@
 .vtex-footer__list-title {
   margin: 0;
   text-transform: uppercase;
+  font-size: 0.875rem;
+}
+
+.vtex-footer__accordion-title {
+  margin: 0;
+  display: inline-block;
```
Comment (ClaudivanFilho) [inline react/components/Footer/global.css]: use tachyons instead

Comment (ClaudivanFilho) [reply react/components/Footer/global.css]: dib ma0 ttu

File: react/components/Footer/global.css [hunk 2]
```diff
@@ -61,27 +73,73 @@
...
+.vtex-footer__links-container {
+  font-size: 0.875rem;
+}
+
+.vtex-footer__links-container .vtex-footer__list {
+  flex-flow: column;
+}
+
+.vtex-footer__matrix-container {
+  display: flex;
```
Comment (ClaudivanFilho) [inline react/components/Footer/global.css]: use tachyons

Comment (ClaudivanFilho) [reply react/components/Footer/global.css]: flex flex-wrap

File: react/components/Footer/global.css [hunk 3]
```diff
@@ -61,27 +73,73 @@
...
+}
+
+.vtex-footer__link-matrix-item--small {
+  border-bottom: 0.5px solid #bdbdbd;
+}
+
 .vtex-footer__list-link {
   display: inline-block;
   text-decoration: none;
+  white-space: nowrap;
```
Comment (ClaudivanFilho) [inline react/components/Footer/global.css]: use `truncate` of tachyons

File: react/components/Footer/global.css [hunk 4]
```diff
@@ -61,27 +73,73 @@
...
 }
 
+/* Mobile */
 @media screen and (min-width: 10em) {
   .vtex-footer__container {
     flex-direction: column;
     padding: 22px 10px;
   }
 
+  .vtex-footer__text-container {
```
Comment (ClaudivanFilho) [inline react/components/Footer/global.css]: use tachyons

File: react/components/Footer/global.css [hunk 5]
```diff
@@ -118,13 +176,16 @@
     padding: 20px;
   }
 
-  .vtex-footer__list {
-    max-height: 150px;
+  .vtex-footer__text-container {
+    width: 80%;
```
Comment (ClaudivanFilho) [inline react/components/Footer/global.css]: tachyons

**Comment by oldufre** [comment]
tu deixasse o fundo cinza, é para deixar branco
footer vanilla 1

**Comment by estacioneto** [comment]
A cor que foi me enviada para o footer foi #E3E4E6, @oldufre

File: react/components/Footer/global.css [hunk 6]
```diff
@@ -1,3 +1,7 @@
+.vtex-footer {
+  box-shadow: 0 -5px 7px -5px #CACBCC;
```
Comment (andreldsa) [inline react/components/Footer/global.css]: You should use the same shadow config as the other elements, like header, e.g.: https://github.com/vtex-apps/store-components/blob/3358dced607722ae61ac93370d2cf974954f9933/react/components/Header/global.css#L14

Attention to the direction of the shadow, once the header points to the bottom.

**Pull request merged**